### PR TITLE
[ntuple] Isolate failing Windows CI TFile test

### PR DESCRIPTION
**Pull request by mxxo**
Try to diagnose problematic test from #7749.

This version of the test checks the success of a minimal ntuple
read-write cycle. If the test fails on CI, we have diagnosed a latent
issue in the RPageSink constructor taking a std::unique_ptr<TFile>&.

### CODE REVIEW

**Comment by mxxo** [comment]
@phsft-bot build just on windows10/cxx14

**Comment by mxxo** [comment]
@phsft-bot build just on windows10/cxx14

**Comment by mxxo** [comment]
@phsft-bot build just on windows10/cxx14

**Comment by mxxo** [comment]
@phsft-bot build just on windows10/cxx14

**Comment by mxxo** [comment]
@phsft-bot build just on windows10/cxx14

File: tree/ntuple/v7/test/ntuple_zip.cxx [hunk 1]
```diff
@@ -84,3 +84,37 @@ TEST(RNTupleZip, Large)
...
+         klass.s = std::to_string(i);
+         *klassVec = {klass};
+         ntuple->Fill();
+      }
+   }
+
+   auto ntuple = RNTupleReader::Open("ntuple", fileGuard.GetPath());
+   std::ostringstream oss;
+   ntuple->PrintInfo(ROOT::Experimental::ENTupleInfo::kStorageDetails, oss);
+   EXPECT_THAT(oss.str(), testing::HasSubstr("Compression: 404"));
```
Comment (bellenot) [inline tree/ntuple/v7/test/ntuple_zip.cxx:112]:
```suggestion
#if !defined(_MSC_VER) || defined(R__ENABLE_BROKEN_WIN_TESTS)
   std::ostringstream oss;
   ntuple->PrintInfo(ROOT::Experimental::ENTupleInfo::kStorageDetails, oss);
   EXPECT_THAT(oss.str(), testing::HasSubstr("Compression: 404"));
#endif
```

**Review by bellenot — APPROVED** [review]

**Pull request closed without merging**